server/routerlicious/packages/services-core/src/mongo.ts

```diff
@@ -89,18 +79,18 @@ export class MongoManager {
             this.reconnect(this.reconnectDelayMs);
         });
 
-        debug("Successfully connected to MongoDB");
-        Lumberjack.info("Successfully connected to MongoDB");
+        debug(`Successfully connected`);
+        Lumberjack.info(`Successfully connected to Db`);
```

Comment:
Nit: why change the quote style here (using \` now) if the original style is still used above (e.g. line 38, `"`)? Should we just keep everything as `"`?

Reply:
I used to have a parameter in the template to indicate which DB is currently used (was useful for debugging), I removed it and forgot to change it back to use `"`.